site/app/libraries/database/DatabaseQueries.php

```diff
@@ -7275,8 +7275,20 @@ public function getCurrentQueue() {
         $this->course_db->query($query, [$this->core->getDateTimeNow()->format('Y-m-d')]);
         return $this->course_db->rows();
     }
-
     public function getPastQueue() {
+        $off  = "ORDER BY ROW_NUMBER";
+        $up   = "ORDER BY helper.user_id";
+        $down = "ORDER BY helper.user_id DESC";
```

Comment:
You could move the "ORDER BY" part to the main body of the SQL query rather than duplicating it here.